lightblue/src/main/java/org/esbtools/eventhandler/lightblue/client/BulkLightblueRequester.java

```diff
@@ -85,7 +85,7 @@ public BulkLightblueRequester(LightblueClient lightblue) {
     }
 
     @Override
-    public TransformableFuture<LightblueDataResponses> request(AbstractLightblueDataRequest... requests) {
+    public TransformableFuture<LightblueDataResponses> request(CRUDRequest... requests) {
```

Comment:
I think it's fine with CRUDRequest, I didn't realize it was required for bulk requests. There is already a generic Requester interface in case another type is needed.